client/src/components/featured.jsx

```diff
@@ -2,16 +2,15 @@ import React from 'react';
 import Review from './review.jsx';
 import moment from 'moment';
 
-const Featured = (props) => {
-  // console.log('Props in Featured:', props);
-  return (
-    <div>
-      {props.review && props.review.reviewer &&
+const Featured = (props) => (
+  <div>
+    {/* {console.log('Props in Featured:', props)} */}
+    {props.review && props.review.reviewer &&
       <div>
         <h2>Featured review</h2>
         <div>
```

Comment:
The html structure from here on down is pretty similar to Review.jsx. Is there a way you can make use of that component rather than copying it's html?